src/components/Heading.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function Heading({ variant, children }) {
+  if (variant === "title") {
+    return <h1 className="heading-1">{children}</h1>;
+  }
+  if (variant === "subtitle") {
+    return <h2 className="heading-2">{children}</h2>;
```

Comment:
Since you have only 2 variants you can use a ternary.
This fixes an issue in your code, currently, not all paths of your component return a value (if the variant passed to component was different than `"title"` or `"subtitle"`, nothing would be returned, like so, you have a fallback)
```suggestion
  return variant === "title"
    ? <h1 className="heading-1">{children}</h1>
    : <h2 className="heading-2">{children}</h2>;
```

Reply:
done, thanks